packages/ibm-products-styles/src/components/Saving/_saving.scss

```diff
@@ -12,6 +12,7 @@
 @use '../../global/styles/project-settings' as c4p-settings;
 
 $block-class: #{c4p-settings.$pkg-prefix}--saving;
+// $carbon-prefix: 'cds';
```

Comment:
Looks like you figured out you could use the `carbon-prefix` from settings. :)
We can remove this line now.